lms/js_config_types.py

```diff
@@ -22,6 +27,8 @@ class APICourse(TypedDict):
     id: int
     title: str
 
+    course_metrics: NotRequired[CourseMetrics]
```

Comment:
Arguably this could be just "metrics" but I reckon in the near feature we'll have annotation_metrics here too.